### DISCUSSION TITLE
Timed Custom Roles

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#225] [Suggestions]
Currently the custom roles only offer a limited amount of time options. 1 day, 1 week, 1 month, and 1 year. 

Please could more options be added to allow communities to offer longer timed roles. For example, if a player makes a higher purchase on support tiers, this could offer them a custom role for longer, giving access to the roles benefits for longer.

Suggested additional options: 7 days, 14 days,   21 days, 2 months, 3months, 4 months, 5 months, 6 months, 7 months, 8 months, 9 months, 10 months, and 11 months.

Thank you,
Gem

**Comment by mvaessen** [comment]
Good point! I suggest we also add the suggested options to:

- Discord roles
- Donor status
- Whitelists

**Reply by newtarkprojects** [reply]
That would be great, thank you Max.

**Comment by Decollates** [comment]
I would also make the suggestion of adding end-of-month expirations.

**Reply by mvaessen** [reply]
great idea!
